lib/Models/ArcGisPortalCatalogGroup.ts

```diff
@@ -317,7 +317,7 @@ export class ArcGisPortalStratum extends LoadableStratum(
       return;
     }
     const id = this._catalogGroup.uniqueId;
-    const itemId = id + "/" + arcgisDataset.id;
+    const itemId = `${id}/${arcgisDataset.id}`;
```

Comment:
It seems there are no fundamental changes here. How does it solve the problem raised in https://github.com/TerriaJS/nsw-digital-twin/issues/459?

Reply:
This change was just for niceness, this is called a [template string](https://developers.google.com/web/updates/2015/01/ES6-Template-Strings), they tend to be a bit nicer that using `"string1" + "string2" + "string3"` pattern

Reply:
I am trying your fixes in my local deployment.

Reply:
Yes, I can confirm that both issues https://github.com/TerriaJS/nsw-digital-twin/issues/458 and https://github.com/TerriaJS/nsw-digital-twin/issues/459 are resolved by this PR. 

However, I don't understand why by checking `arcgisPortalItem.extent` can solve items missing in catalog problem. Is there anything I miss?

Reply:
So I think the extent can also be retrieved via the `Cesium3DTilesCatalogItem` which reads it from the `SceneServer` endpoint rather than the portal item. So I suspect the dodgy value was overriding the correct value.